boards/sparkfun-thing-plus-rp2040/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+#![no_std]
+
+pub use rp2040_hal as hal;
+#[cfg(feature = "rt")]
+extern crate cortex_m_rt;
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+/// The linker will place this boot block at the start of our program image. We
+/// need this to help the ROM bootloader get our code up and running.
+#[cfg(feature = "boot2")]
+#[link_section = ".boot2"]
+#[no_mangle]
+#[used]
+pub static BOOT2_FIRMWARE: [u8; 256] = rp2040_boot2::BOOT_LOADER_W25Q080;
+
+pub use hal::pac;
+
+hal::bsp_pins!(
+    Gpio0 { name: tx },
+    Gpio1 { name: rx },
+    Gpio2 { name: sck },
+    Gpio3 { name: copi },
+    Gpio4 { name: cipo },
+    Gpio6 { name: sda },
+    Gpio7 { name: scl },
+    Gpio8 { name: ws2812 },
+    Gpio16 { name: gpio16 },
+    Gpio17 { name: gpio17 },
+    Gpio18 { name: gpio18 },
+    Gpio19 { name: gpio19 },
+    Gpio20 { name: gpio20 },
+    Gpio21 { name: gpio21 },
+    Gpio22 { name: gpio22 },
+    Gpio25 { name: led_builtin },
```

Comment:
Could you rename Gpio25 from `led_builtin` to `led`?
Where possible I'm trying to keep the [template blinky code](https://github.com/rp-rs/rp2040-project-template/blob/f4fd405c917ee4d077f70bff63c46b76f6db1838/src/main.rs#L57) portable across BSPs, which is easier if it has the same name.

Reply:
Can do. I used `led_builtin` to match SparkFun's schematic, which I think they did to disambiguate this LED from the WS2812, but keeping the names consistent makes sense to me too.